tox.ini

```diff
@@ -3,6 +3,7 @@
 envlist=flake8,docs,py27
 
 [testenv]
+whitelist_externals = make
```

Comment:
Inherited from Tadej's PR. Seems to be suppressing a warning when the `docs` env runs make (so I'll drop it in the docs PR because we'll be running `sphinxify-build` instead)